lib/azkaban_scheduler/client.rb

```diff
@@ -10,6 +10,7 @@ def initialize(url, http_options = {}, client_headers={})
       @client_headers = client_headers
 
       @http = Net::HTTP.new(uri.host, uri.port)
+      @http.read_timeout = 180
```

Comment:
This seems like an application specific configuration change.  It seems like this should be done through `http_options` instead rather than making releases just to change a default.

Reply:
That's a great point. I'll throw up another PR with the change.

Reply:
Oh I didn't even realize you could pass it in as an option already, that was silly. I'm gonna revert the PR